app/services/imports/sales_logs_field_import_service.rb

```diff
@@ -28,5 +32,25 @@ def update_creation_method(xml_doc)
         @logger.info "sales log #{log.id} creation method set to bulk upload"
       end
     end
+
+    def update_owning_organisation_id(xml_doc)
+      return if meta_field_value(xml_doc, "form-name").include?("Lettings")
+
+      old_id = meta_field_value(xml_doc, "document-id")
+      record = SalesLog.find_by(old_id:)
+
+      if record.present?
+        if record.owning_organisation_id.present?
+          @logger.info("sales log #{record.id} has a value for owning_organisation_id, skipping update")
+        else
+          old_owning_organisation_id = safe_string_as_integer(xml_doc, "OWNINGORGID")
```

Comment:
`find_organisation_id(xml_doc, "OWNINGORGID")` ?

Reply:
Or `string_or_nil(xml_doc, id_field)` if you're trying to recreate the `find_organisation_id` method

Reply:
🙌